src/components/projects-summary/projects-summary.controller.ts

```diff
@@ -167,4 +236,20 @@ export class ProjectsSummaryController implements angular.IController {
     $('.catalog-projects-summary-modal-backrop').remove();
   }
 
+  private getRecentlyViewedItems() {
+    // recentItems is an array of uids
+    let recentItems: any = this.RecentlyViewed.getItems();
+
+    // replace uids with IServiceItems
+    let items: any = _.map(recentItems, (uid: any) => {
+      return this.allItems[uid];
```

Comment:
Missed this on previous review, but we might want a null check here because this will be a runtime error if you haven't passed in the items. Do we want to require that for this component?